packages/ag-charts-enterprise/src/features/annotations/parallel-channel/parallelChannelScene.ts

```diff
@@ -150,6 +150,17 @@ export class ParallelChannelScene extends ChannelScene<ParallelChannelProperties
         return [bottomLeft, bottomRight];
     }
 
+    override copy(
+        datum: ParallelChannelProperties,
+        copiedDatum: ParallelChannelProperties,
+        context: AnnotationContext,
+        offset: Coords
+    ) {
+        super.copy(datum, copiedDatum, context, offset);
+        copiedDatum.height = datum.height;
```

Comment:
I think this is covered by the `copiedDatum.set(datum.getJson()` call previously? In which case this whole override can be deleted?